### PR TITLE
Carousel cards modified - responsive

### PR DESCRIPTION
**Pull request by florinpeslar**

### CODE REVIEW

File: src/components/features/services-list/ServicesList.vue [hunk 1]
```diff
@@ -22,15 +22,24 @@
...
+        <div v-else class="col-12 owl-carousel service-carousel owl-theme categories_carousel_in">
+          <services-list-item
+            v-for="item in service.items"
+            :image-path="getImagePath(item)"
+            :key="item.category"
+            :service="item"
+            :to="getToRoute(item)"
+          />
+        </div>
+        <!-- <template v-else>
```
Comment (norbertdemeny13) [inline src/components/features/services-list/ServicesList.vue:34]: remove if not needed

File: src/vendors/common_func.js [hunk 2]
```diff
@@ -220,6 +220,69 @@
     });
   };
 
+  window.promoCarousel = function() {
```
Comment (norbertdemeny13) [inline src/vendors/common_func.js:223]: Why do you defined this here and in the common_func as well? Use common_func for general functions such as window object. Vue component should contain only vue elements

File: src/vendors/common_func.js [hunk 3]
```diff
@@ -220,6 +220,69 @@
...
+        },
+        1230: {
+          items: 3,
+          nav: true
+        }
+      },
+    });
+  };
+
+  window.serviceCarousel = function() {
```
Comment (norbertdemeny13) [inline src/vendors/common_func.js:256]: same here

File: src/vendors/common_func.js (hunk 2 above)
Comment (florinpeslar) [reply src/vendors/common_func.js:223]: This approach was used in other vue components too. I needed as general rule to have access to customize only some specific carousels.

**Pull request merged**